linux_sql/README.md

```diff
@@ -1,2 +1,38 @@
-#Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Review.
+# <h1>Linux Cluster Monitoring Agent
```

Comment:
please remove all HTML tags such as `<h1>` ..you can edit README file from IDE or any markdown editor (such as Typora)